cli/src/commands/shared_recovery/create.rs

```diff
@@ -93,6 +99,29 @@ pub async fn create_shared_recovery(args: Args, client: &StartedClient) -> anyho
             per_recipient_shares.len()
         )) .interact_text()?
     };
+
+    println!(
+        "The following shared recovery setup will be created:\n{BULLET_CHAR} Threshold: {threshold}\n{}",
+        per_recipient_shares
+            .iter()
+            .map(|(recipient, share)| {
+                let user = &users
+                    .iter()
+                    .find(|x| x.id == *recipient)
+                    .expect("missing recipient")
+                    .human_handle;
+                format!("{BULLET_CHAR} User {user} will have {share} share(s)") // TODO: special case if there is only one share
```

Comment:
Maybe treat this now since it's simple enough?

Reply:
It's in quite a few other places too.